protocols/v2/subprotocols/common-messages/src/setup_connection.rs

```diff
@@ -391,13 +406,31 @@ mod test {
     #[test]
     fn test_check_flag() {
         let protocol = crate::Protocol::MiningProtocol;
-        let flag_available = 0b_0000_0000_0000_0000_0000_0000_0000_0000;
+        let flag_avaiable = 0b_0000_0000_0000_0000_0000_0000_0000_0000;
         let flag_required = 0b_0000_0000_0000_0000_0000_0000_0000_0001;
         assert!(SetupConnection::check_flags(
             protocol,
-            flag_available,
+            flag_avaiable,
```

Comment:
it seems we're changing this line just to introduce a typo?

Reply:
Will correct it out